<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Video Downloader Y-V-D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icon.png</normaloff>icons/icon.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QPushButton" name="download_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>600</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_url">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>YouTube URL: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_url">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>591</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>YouTube link ex: https//www.youtube.com/watch?v=example</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_res">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>170</y>
     <width>68</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="combobox_res">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1080p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>360p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>144p</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>230</y>
     <width>136</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressbar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>551</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_statue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Statue: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="le_statue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>831</width>
     <height>161</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>For any questions, visit me on: https://exzandar.home.blog/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;YVD is a simple app to download videos and audios from YouTube with the wanted quality!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;It downloads the video then the audio and merge them together.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_author">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author: @exzandar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file_path">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>561</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Click on Browse to choose where you want to save your file.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_f_num">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>311</y>
     <width>74</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>file number: </string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>311</y>
     <width>64</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_from_num">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>310</y>
     <width>36</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>from: </string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>311</y>
     <width>64</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>631</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QRadioButton" name="rd_a">
    <property name="text">
     <string>Audio only</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_v">
    <property name="text">
     <string>Video only</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_c">
    <property name="text">
     <string>Audio and Video in one clip</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_ac">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Two files, clip+audio</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Search for results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tst_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Button</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_url</tabstop>
  <tabstop>combobox_res</tabstop>
  <tabstop>rd_a</tabstop>
  <tabstop>rd_v</tabstop>
  <tabstop>rd_c</tabstop>
  <tabstop>rd_ac</tabstop>
  <tabstop>save_btn</tabstop>
  <tabstop>le_statue</tabstop>
  <tabstop>download_btn</tabstop>
  <tabstop>exit_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save_btn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>browse_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>download_btn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>prog_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_btn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>get_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tst_btn</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>test_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_file()</slot>
  <slot>download()</slot>
  <slot>prog_exit()</slot>
  <slot>get_data()</slot>
  <slot>test_btn()</slot>
 </slots>
</ui>
